src/libmongoc/tests/json-test-monitoring.c

```diff
@@ -262,6 +262,7 @@ set_apm_callbacks (json_test_ctx_t *ctx, mongoc_client_t *client)
 {
    mongoc_apm_callbacks_t *callbacks;
 
+   ASSERT (client);
```

Comment:
`client` is unused in this function. Suggest removing the `mongoc_client_t *client` argument.